### DISCUSSION TITLE
How I can integrate the the Multi-Step Element in another Element?

### DISCUSSION THREAD

**Discussion by silviaAut** [formkit/formkit#738] [General]
I want to achieve that:
![image](https://github.com/formkit/formkit/assets/6195977/3af5f5eb-8cf7-4ef0-9e76-1c7f7e8d296f)

How can I integrate the Steps Markup into another Component?

Best regards,
Silvia

**Comment by silviaAut** [comment]
Edit: I use this Plugin: https://formkit.com/plugins/multi-step

**Comment by andrew-boyd** [comment]
Splitting the tab navigation away from the step content is not currently a supported feature of the multi-step. In the future maybe we can provide a separate `tabs` input type and then you'd be able to build your own UI by reacting to the `tabs` input value to conditionally show / hide different form content.
